### ISSUE TITLE
Change to async_forward_entry_setups

### ISSUE THREAD

**Issue by Snuffy2** [prestomation/resmed_myair_sensors#30]
## Describe the bug
```
2023-02-01 21:06:44.621 WARNING (MainThread) [homeassistant.helpers.frame] Detected integration that called async_setup_platforms instead of awaiting async_forward_entry_setups; this will fail in version 2023.3. Please report issue to the custom integration author for resmed_myair using this method at custom_components/resmed_myair/__init__.py, line 23: hass.config_entries.async_setup_platforms(config_entry, PLATFORMS)
```

Linked PR to fix

**Comment by Snuffy2** [comment]
https://developers.home-assistant.io/blog/2022/07/08/config_entry_forwards/

**Comment by keisisqrl** [comment]
Home Assistant now reports this may fail in current versions:

2023-03-21 14:02:22.920 WARNING (MainThread) [homeassistant.helpers.frame] Detected integration that called async_setup_platforms instead of awaiting async_forward_entry_setups; this will fail in version 2023.3. Please report issue to the custom integration author for resmed_myair using this method at custom_components/resmed_myair/__init__.py, line 23: hass.config_entries.async_setup_platforms(config_entry, PLATFORMS)

**Comment by Madj42** [comment]
@prestomation I know you don't have the device any longer and you no longer maintain the code.  Any chance this and the other PR could be merged into a beta so we could try this out?  Thanks.

**Comment by prestomation** [comment]
fix with #29
